UX4_ValidityReliability.ipynb

```diff
@@ -663,21 +669,21 @@
     }
    ],
    "source": [
-    "# logarithmically transformed values\n",
```

Comment:
I don't feel like Logarithmically transformed values has been well introduced in the notebooks, and it is hard to see the difference in the graph. Maybe add a sentence that highlights what it means?

Reply:
True. Actually, this is the first version and I anticipate I'll have to tweak things for next year. But I don't have much time each week for this and it does take time to organize the material in a good way... For now, I'll handle this in class.